### ISSUE TITLE
Icon Request: gnome-books

### ISSUE THREAD

**Issue by likeaross** [numixproject/numix-core#3576]
Desktop file:

`/usr/share/applications/org.gnome.Books.desktop`

```
[Desktop Entry]
Name=Books
Comment=Access, manage and share books
Exec=gnome-books
Icon=org.gnome.Books
Type=Application
OnlyShowIn=GNOME;
Categories=GNOME;GTK;Utility;Core;
Keywords[en_GB]=Books;Comics;ePub;PDF;
```
Original Icon:
![org gnome books](https://cloud.githubusercontent.com/assets/483960/22570869/6c23b6e2-e95a-11e6-88d5-6a1e5f70568e.png)

Seems to be installed with evince as far as I can tell

**Comment by bilelmoussaoui** [comment]
We already have that icon https://github.com/numixproject/numix-core/blob/master/data.json#L5152. I guess that you're using an outdated version of Numix Circle theme.  Duplicate of https://github.com/numixproject/numix-core/issues/1800

**Comment by likeaross** [comment]
Ahh. I see the icon with Numix Circle, however it is not currently showing with the square theme.
I have the icon theme installed from this [AUR](https://aur.archlinux.org/packages/numix-icon-theme-pack/) repository.

**Comment by bilelmoussaoui** [comment]
Well, the Square version is still not as complete as Circle, this should be fixed once those issues are fixed #3481

**Comment by likeaross** [comment]
I see, thank you for the information.
